src/Uno.UI/UI/Xaml/Controls/TreeView/TreeViewViewModel.cs

```diff
@@ -0,0 +1,869 @@
+
+			//    var selectedItems = make_self<SelectedItemsVector>();
+			//selectedItems.SetViewModel(*this);
+			//m_selectedItems.set(* selectedItems);
+
+			//    m_itemToNodeMap.set(make<HashMap<IInspectable, TreeViewNode>>());
+		}
+
+		//ViewModel.~ViewModel()
+		//{
+		//	if (m_rootNodeChildrenChangedEventToken.value != 0)
+		//	{
+		//		if (var origin = m_originNode.safe_get())
+		//        {
+		//			get_self<TreeViewNode>(origin).ChildrenChanged(m_rootNodeChildrenChangedEventToken);
+		//		}
+
+		//		ClearEventTokenVectors();
+		//	}
+		//}
+
+		internal void ExpandNode(TreeViewNode value)
+		{
+			value.IsExpanded = true;
+		}
+
+		internal void CollapseNode(TreeViewNode value)
+		{
+			value.IsExpanded = false;
+		}
+
+		//event_token ViewModel.NodeExpanding(const TypedEventHandler<TreeViewNode, IInspectable>& handler)
+		//{
+		//	return m_nodeExpandingEventSource.add(handler);
+		//}
+
+		//void ViewModel.NodeExpanding(const event_token token)
+		//{
+		//	m_nodeExpandingEventSource.remove(token);
+		//}
+
+		//event_token ViewModel.NodeCollapsed(const TypedEventHandler<TreeViewNode, IInspectable>& handler)
+		//{
+		//	return m_nodeCollapsedEventSource.add(handler);
+		//}
+
+		//void ViewModel.NodeCollapsed(const event_token token)
+		//{
+		//	m_nodeCollapsedEventSource.remove(token);
+		//}
+
+		internal void SelectAll()
+		{
+			UpdateSelection(m_originNode, TreeNodeSelectionState.Selected);
+		}
+
+		private void ModifySelectByIndex(int index, TreeNodeSelectionState state)
+		{
+			var targetNode = GetNodeAt(index);
+			UpdateSelection(targetNode, state);
+		}
+
+		internal int Size
+		{
+			get
+			{
+				//	var inner = GetVectorInnerImpl();
+				//	return inner.Size();
+				throw new NotImplementedException();
+			}
+		}
+
+		private object GetAt(uint index)
+		{
+			//TreeViewNode node = GetNodeAt(index);
+			//return IsContentMode() ? node.Content() : node;
+			throw new NotImplementedException();
+		}
+
+		private uint IndexOf(object value, uint index)
+		{
+			//	if (var indexOfFunction = GetCustomIndexOfFunction())
+			//    {
+			//		return indexOfFunction(value, index);
+			//	}
+
+			//	else
+			//	{
+			//		var inner = GetVectorInnerImpl();
+			//		return inner.IndexOf(value, index);
+			//	}
+			throw new NotImplementedException();
+		}
+
+		//uint32_t ViewModel.GetMany(uint32_t const startIndex, array_view<IInspectable> values)
+		//{
+		//	var inner = GetVectorInnerImpl();
+		//	if (IsContentMode())
+		//	{
+		//		var vector = make<Vector<IInspectable>>();
+		//		int size = Size();
+		//		for (int i = 0; i < size; i++)
+		//		{
+		//			vector.Append(GetNodeAt(i).Content());
+		//		}
+		//		return vector.GetMany(startIndex, values);
+		//	}
+		//	return inner.GetMany(startIndex, values);
+		//}
+
+		//IVectorView<IInspectable> ViewModel.GetView()
+		//{
+		//	throw hresult_not_implemented();
+		//}
+
+		internal TreeViewNode GetNodeAt(int index)
+		{
+			//	var inner = GetVectorInnerImpl();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5156819)